bindings/python/pymongocrypt/credentials.py

```diff
@@ -63,7 +63,7 @@ def _get_azure_credentials():
     # Credentials are considered expired when: Expiration - now < 1 mins.
     creds = _azure_creds_cache
     if creds:
-        if creds.expires_in - datetime.utcnow() < timedelta(seconds=60):
+        if creds.expires_in - datetime.now(tz=timezone.utc) < timedelta(seconds=60):
```

Comment:
Won't this now fail because creds.expires_in is naive and `datetime.now(tz=timezone.utc)` is aware?

Reply:
`expires_in` is also using `utc`, I renamed the variable to `expires_utc` for clarity.